TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/colorSensor/colorSensorTest.java

```diff
@@ -0,0 +1,23 @@
+package org.firstinspires.ftc.teamcode.auton.colorSensor;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+
+public class colorSensorTest extends LinearOpMode {
+    ColorSensor color_sensor;
+
+    public void runOpMode() {
+        color_sensor = hardwareMap.colorSensor.get("color");
+
+        waitForStart();
+
+        while (opModeIsActive()) {
+            //finds amount of red, green, blue, and light (alpha)
+            telemetry.addData("red: ", color_sensor.red());
```

Comment:
did you write these methods yet? or are they already defined in the color sensor? If so, let's figure out a way to run a method that allows us to customize threshold and use custom colors (Scalars, i think).